roles/backed-up/files/backup-mysql

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -eo pipefail
+/usr/bin/mysqldump --defaults-extra-file=/usr/local/etc/mysql/backup.cnf --events --ignore-table=mysql.event --all-databases --skip-extended-insert --single-transaction | bzip2 -9 - > /var/lib/backup/mysql.bz2.$$
```

Comment:
Are you expecting to have multiple mysqldumps in `$PATH`?

Reply:
This partially came from https://github.com/sown/ansible-playbooks/blob/master/files/mysql/mysqlbackup - based on how it used to run off cron, I suspect it was the other way round, and /usr/bin was sometimes missing from PATH entirely.

mysqldump is in the PATH when running under ssh like we do now though, so I'll drop this to just calling `mysqldump`.